### ISSUE TITLE
(very minor) "Manage Mod" button gets placed strangely on mods where you are not an owner on certain devices/breakpoints/browsers

### ISSUE THREAD

**Issue by Thallassa** [Nexus-Mods/web-issues#285]
A user posted this in mod author discord earlier: 

https://cdn.discordapp.com/attachments/232170227734020097/374644347590672404/image.png (not my image)

I immediately noticed the strange placement of the buttons (which was not why he posted it). 

However I was not able to replicate this at any breakpoint nor on my android phone. So I'm not really sure what's going on there. 

ETA: It seems that there's an issue with the placement of the manage buttons on mods where you have management rights, but are not the file owner. 

Same user, but on his *own* mod: https://cdn.discordapp.com/attachments/232170227734020097/374651811140730890/image.png

My mod:

https://cdn.discordapp.com/attachments/232170227734020097/374652147469385728/unknown.png

Mod I have management rights on:

https://cdn.discordapp.com/attachments/232170227734020097/374652015512387603/unknown.png

tl;dr On mods you can manage, but do not own, the "manage" button gets placed strangely on *very* specific breakpoints/browser sizes.

**Comment by iamtiz** [comment]
assigned #155431824

**Comment by iamtiz** [comment]
done
